TRACK_README.md

```diff
@@ -35,7 +35,7 @@ Please keep the following in mind:
 - Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
 [exercism/x-common](https://github.com/exercism/x-common) repository. Further explanation can be found in [fixing-exercise-readmes](https://github.com/exercism/exercism.io/blob/master/docs/fixing-exercise-readmes.md)
 
-- Each exercise should have a test suite, an example solution, a template file for the real implementation and ... (anything else that needs to go with each exercise for this track). The CI build expects files to be named using the following convention. The example solution should be named `ExerciseModuleName.example`. The template file should be named `ExerciseModuleName.{{LANGUAGE}}`. Test file should be named `ExerciseModuleNameTest.{{LANGUAGE}}`. (Use the naming conventions of the track's language.)
+- Each exercise should have a test suite, an example solution, a template file for the real implementation and ... (anything else that needs to go with each exercise for this track). The CI build expects files to be named using the following convention. The example solution should be named `{{LANGUAGE specific naming convention e.g. ExerciseModuleName.example}}`. The template file should be named `{{e.g. ExerciseModuleName.LANGUAGE}}`. Test file should be named `{{e.g. ExerciseModuleNameTest.LANGUAGE}}`. (Use the naming conventions of the track's language.)
```

Comment:
The replacement is `{{LANGUAGE}}`. Anything else in the brackets will cause the replacement to fail